code_samples/api/commerce/src/Command/PaymentCommand.php

```diff
@@ -67,6 +71,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $paymentIdentifier = '4ac4b8a0-eed8-496d-87d9-32a960a10629';
         $payment = $this->paymentService->getPaymentByIdentifier($paymentIdentifier);
 
+        $context = $payment->getContext();
+
```

Comment:
`$context` seems not used between this set and its override at line 97 (below `// Create a new payment`)

Reply:
You're right, it doesn't make a lot of sense in the context of the Command - I've included it to improve the doc a bit:

Please compare:
https://ez-systems-developer-documentation--2748.com.readthedocs.build/en/2748/commerce/payment/extend_payment/#attach-custom-data-to-payments
and
https://doc.ibexa.co/en/latest/commerce/payment/extend_payment/#attach-custom-data-to-payments

I could add the `getContext` line somewhere else and then "glue" 2 includes together, I just prefered having it in one include as I think it's easier to maintain

Reply:
That's tricky!
I didn't see that `extend_payment.md` uses it in reverse order…
Maybe a comment in the cut-out line to explain why it's there?
```suggestion
        $context = $payment->getContext();
        // Will be overridden later but used to illustrate `getContext()`
```

Reply:
Added in https://github.com/ibexa/documentation-developer/pull/2748/commits/79c5c1af91bcfc980c5ecbed8a3305d91d772d7a - I'll wait for the "check code samples" job to make sure I did it right